packages/salesforcedx-vscode-apex-debugger/src/index.ts

```diff
@@ -31,7 +31,22 @@ function registerCommands(): vscode.Disposable {
       );
     }
   );
-  return vscode.Disposable.from(initialDebugConfig);
+  const getLineBreakpointInfo = vscode.debug.onDidStartDebugSession(
```

Comment:
vscode-apex-debugger listens to a session start event. If triggered, use vscode.extensions namespace to ask vscode-apex module for line breakpoint info. Return this info to apex debugger adapter.